src/components/input/UIInput.vue

```diff
@@ -117,12 +117,20 @@
             </div>
         </div>
 
-        <UIExpandTransition>
-            <slot v-if="error || $slots.error" name="error">
-                <p class="text-color-error text-sm">
-                    {{ error }}
-                </p>
-            </slot>
+        <UIExpandTransition :height-duration="100" :opacity-duration="200">
+            <template v-if="error || $slots.error || help || $slots.help">
+                <slot v-if="error || $slots.error" name="error">
+                    <p class="ui-error-text">
+                        {{ error }}
+                    </p>
+                </slot>
+
+                <slot v-else-if="(help || $slots.help)" name="help">
+                    <p class="ui-help-text">
+                        {{ help }}
+                    </p>
+                </slot>
+            </template>
```

Comment:
sorry to be a pain but what do you think about this?
```suggestion
            <div v-if="error || $slots.error || help || $slots.help">
                <UIFadeTransition>
                    <slot v-if="error || $slots.error" name="error">
                        <p class="ui-error-text">
                            {{ error }}
                        </p>
                    </slot>

                    <slot v-else-if="(help || $slots.help)" name="help">
                        <p class="ui-help-text">
                            {{ help }}
                        </p>
                    </slot>
                </UIFadeTransition>
            </div>
```
cc @fullstackfool